playground/AspireEventHub/EventHubs.AppHost/Program.cs

```diff
@@ -0,0 +1,19 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
+
+// required for the event processor client which will use the connectionName to get the connectionString.
+var blob = builder.AddAzureStorage("ehstorage")
+    .AddBlobs("checkpoints");
+
+var eventHub = builder.AddAzureEventHubs("eventhubns")
+    .AddEventHub("hub");
```

Comment:
@jsquire - should this be:

```C#
var eventHub = builder.AddAzureEventHubNamespace("eventhubns")
    .AddEventHub("hub");
```

Looking at the portal and in the bicep, I think this top level thing is called a "Namespace". Should we be reflecting that in the API?

Reply:
I think we should file a follow up to do this for both service bus and this resource.

Reply:
Technically, yes.   You're correct about the nomenclature.  I do think it adds a bit of clarity, though I didn't find it difficult to understand in the current form, either.

Reply:
I think that would be good, but as others have said, ASB should reflect the same nomenclature. As it stands, I'm doing what the service bus stuff does.